ArduCopter/APM_Config.h

```diff
@@ -42,6 +42,7 @@
 //#define MODE_SMARTRTL_ENABLED DISABLED            // disable smartrtl mode support
 //#define MODE_SPORT_ENABLED DISABLED               // disable sport mode support
 //#define MODE_THROW_ENABLED    DISABLED            // disable throw mode support
+//#define DEVO_TELEM_ENABLED DISABLED
```

Comment:
Please add a small comment explaining a bit what is DEVO telemetry

Reply:
Done